test/TestRRUtils.sol

```diff
@@ -64,4 +65,15 @@ contract TestRRUtils {
     // Does not exist, past the end of the bitmap windows
     Assert.equal(tb.checkTypeBitmap(1, 1281), false, "Type 1281 should not exist in type bitmap");
   }
+
+  function testCompareLabel() public {
```

Comment:
This definitely needs to test a few more cases:

 - When one name has a difference of >1 label to the other (a.b.c vs c)
 - When two names start the same but have differences in later labels (a.b.c vs a.d.c)
 - When the first label sorts later, but the first label sorts earlier (a.b.c vs b.a.c)
 - Two names where the first label on one is a prefix of the first label on the other (ab.c.d vs a.c.d)